device/nvidia-bluefield/arm64-nvda_bf-9009d3b600cvaa/platform.json

```diff
@@ -0,0 +1,36 @@
+{
+    "chassis": {
+        "name": "Nvidia-9009d3b600CVAA",
+        "components": [],
+        "fans": [],
+        "fan_drawers": [],
+        "psus": [],
+        "thermals": [],
+        "sfps": [
+            {
+                "name": "Ethernet0"
+            },
+            {
+                "name": "Ethernet4"
+            }
+        ]
+    },
+    "interfaces": {
+        "Ethernet0": {
+            "index": "1,1,1,1",
+            "lanes": "0,1,2,3",
+            "breakout_modes": {
+                "1x100G": ["etp1"],
```

Comment:
Can't we have one line "1x100G[200G]" : ["etp1"]

Reply:
No, this will have a different meaning.
The DPU ports should support 100G or 200G.
`1x100G[200G]` means that port supports both 100G and 200G